### PR TITLE
Added docstrings to Model.py -> Partial fix to #19

### PR DESCRIPTION
**Pull request by caidol**

### CODE REVIEW

File: Models.py [hunk 1]
```diff
@@ -195,6 +228,18 @@ def load(self) -> Optional[Self]:
...
+        This method will return the indicators for the past `num_days` days specified in the information keys. 
+        It will use the cached information to calculate the indicators until the `end_date`.
+
+        Args:
+            cached_info (pd.DataFrame): The cached information
+            end_date (datetime): The end date
+            num_days (int): The number of days to calculate the indicators for
+        
+        Returns:
+            stock_data (dict): A dictionary containing the indicators for the stock data
```
Comment (gran4) [inline Models.py:241]: should just be `dict: `

File: Models.py [hunk 2]
```diff
@@ -270,6 +315,18 @@ def indicators_past_num_days(self, cached_info, end_date, num_days):
...
+        This method calculates the indicators for the stock data for the current day. 
+        It will use the current day_info to calculate the indicators until the `end_date`.
+
+        Args:
+            day_info (pd.DataFrame): The stock data for the current day
+            end_date (datetime): The date to stop calculating the indicators
+            num_days (int): The number of days to calculate the indicators for
+        
+        Returns: 
+            stock_data (dict): A dictionary of the indicators for the stock data
```
Comment (gran4) [inline Models.py:328]: Same as above

**Review by gran4 — CHANGES_REQUESTED** [review]
Sorry I am so late, I was busy all day.

**Comment by caidol** [comment]
> Sorry I am so late, I was busy all day.

No worries I understand.


> Thanks again

No problem! I should have fixed the changes so it'll be good to merge into Models.py now.

**Pull request merged**